installer/test/phx_new_test.exs

```diff
@@ -191,6 +191,29 @@ defmodule Mix.Tasks.Phx.NewTest do
         assert file =~ "summary(\"phx_blog.repo.query.total_time\","
       end
 
+      # Mailer
+      assert_file "phx_blog/mix.exs", fn file ->
+        assert file =~ "{:swoosh, \"~> 1.3\"}"
+      end
+
+      assert_file "phx_blog/lib/phx_blog/mailer.ex", fn file ->
+        assert file =~ "defmodule PhxBlog.Mailer do"
+        assert file =~ "use Swoosh.Mailer, otp_app: :phx_blog"
+      end
+
+      assert_file "phx_blog/config/config.exs", fn file ->
+        assert file =~ "config :swoosh"
+        assert file =~ "config :phx_blog, PhxBlogWeb.Mailer, adapter: Swoosh.Adapters.SMTP"
```

Comment:
Is this an oversight? I would expect the namespace to be ˋPhxBlogˋ and not ˋ PhxBlogWebˋ given thats the module asserted for in line 200 of this test.

Reply:
Seems like this has been fixed in a followup PR.